### PR TITLE
Promote Awesome

### PR DESCRIPTION
**Pull request by ozh**
I'm adding a "YOURLS Awesome" button on all listed repo.

Regarding this one specifically I have no idea how this thing works, I'll let you decide @LeoColomb :)

### CODE REVIEW

**Comment by LeoColomb** [comment]
Indeed, it is a good idea to include a badge for this.
But why placing the badge in the title and not with the other badges?

**Comment by ozh** [comment]
> But why placing the badge in the title and not with the other badges?

Absolutely. I was in a mass editing spree and went for the same placement everywhere 🙄

File: README.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
-# YOURLS using Helm [![Listed in Awesome YOURLS!](https://img.shields.io/badge/Awesome-YOURLS-C5A3BE)](https://github.com/YOURLS/awesome-yourls/)
-
+# YOURLS using Helm
```
Comment (LeoColomb) [inline README.md:1]:
```suggestion
# YOURLS using Helm

```

Dropping the added space, and re-adding the deleted line.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
-# YOURLS using Helm [![Listed in Awesome YOURLS!](https://img.shields.io/badge/Awesome-YOURLS-C5A3BE)](https://github.com/YOURLS/awesome-yourls/)
-
+# YOURLS using Helm 
 > The official [Helm](https://helm.sh) charts repository for [YOURLS](https://yourls.org).
 
 [![Build Status](https://github.com/YOURLS/charts/workflows/Charts%20CI/badge.svg)](https://github.com/YOURLS/charts/actions)
 [![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/yourls)](https://artifacthub.io/packages/search?repo=yourls)
 [![Chart Version](https://img.shields.io/badge/dynamic/yaml.svg?color=blue&label=chart&prefix=v&query=entries.yourls%5B0%5D.version&url=https%3A%2F%2Fcharts.yourls.org%2Findex.yaml)](https://artifacthub.io/packages/helm/yourls/yourls)
+[![Listed in Awesome YOURLS!](https://img.shields.io/badge/Awesome-YOURLS-C5A3BE)](https://github.com/YOURLS/awesome-yourls/)
+
```
Comment (LeoColomb) [inline README.md:8]:
```suggestion
[![Listed in Awesome YOURLS](https://img.shields.io/badge/Awesome-YOURLS-C5A3BE)](https://github.com/YOURLS/awesome-yourls/)
```

Dropping the extra line, and removing the exclamation mark (let's keep the tone formal 😊)

**Review by LeoColomb — CHANGES_REQUESTED** [review]

**Review by LeoColomb — APPROVED** [review]

**Pull request merged**